File: examples/c-config/src/show-config.c
#include <stdlib.h>
#include <stdio.h>
#include <unistd.h>

#include "config.h"
#include "config.c"

int main() 
{
   const struct config_content* c = config_get_content ("example.conf");
   if (c == NULL) {
      printf ("FAIL: No configuration file 'example.conf'\n");
      return 1;
   }

   return (write (STDOUT_FILENO, c->content, c->size) == c->size) ? 0 : 1;
}

File: examples/go-config/show-config.go
package main

import (
   "fmt"
   "go-config/config"
)


func main() {
   c := config.Get_content("example.conf")
   if (c == nil) {
      fmt.Println("FAIL: No configuration file 'example.conf'")
      return
   }

   fmt.Println(string(c.Content))
}
